### DISCUSSION TITLE
backup fails on path

### DISCUSSION THREAD

**Discussion by bassmannate** [nextcloud/all-in-one#3193] [Questions] [answered]
I've got a fresh install and I'm trying to set up back ups from AIO. It appears the issue is that it doesn't see my path as valid but I copied it straight from the terminal on my host. 

I see that it uses sshfs from the container to the host to simplify specifying the path but I don't understand how the container would know what my user name or password or even a key-pair would be for authentication. Could this be the issue?

Could someone help a noob out? Everything else about AIO works great but I don't like the idea of really putting a whole lot of data into it before I have a good back up solution.

Edit: Sorry, I should have mentioned this is on OpenMediaVault. I installed AIO via portainer.

Another edit: Here's the error I dug out of the log. The log truncated my path on the host but I don't think it truncated it in execution? I think that just happened because the log line was too long?

`Message: Server error: `POST http://localhost/v1.41/containers/create?name=nextcloud-aio-borgbackup` resulted in a `500 Internal Server Error` response:
{&quot;message&quot;:&quot;invalid volume specification: &#039;/run/user/1000/gvfs/sftp:host=192.168.1.224/srv/dev-disk-by-uuid-06b0856b-471 (truncated...)
File: /var/www/docker-aio/php/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php`

**Comment by szaimen** [comment]
Hi can you be a bit more verbose on what you did? Did you create a doxker volume with docker volume create and pit in your credentials there or is the sshfs a host mount that is moubted via fstab for example?

**Reply by bassmannate** [reply]
I haven't done anything except install and point the backup utility at a directory on the host. Where would it be looking for credentials if I mounted a volume? Is it just looking for an ssh key? Is there some instructions I missed?

**Reply by szaimen** [reply]
I see. So I suppose you've mounted the sshfs then via your host in the location that you've entered in the AIO interface?

**Reply by bassmannate** [reply]
No, I haven't mounted anything in the host. I put a path into the backup utility that already exists local to the host. It looks like the container is trying to mount the local host location over sshfs. I could mount a bind location but I think the utility is programmed to mount everything as remote.

**Reply by szaimen** [reply]
How did you install docker on omv? Is docker installed natively or inside a VM?

**Reply by bassmannate** [reply]
It is installed natively.

**Comment by bassmannate** [comment, marked as answer]
Nevermind. I'm an idiot. I forgot I was copying a path on my laptop that was connected by sshfs. The path was so long that I didn't see that the sshfs stuff was at the beginning of what I was pasting. I assumed it was being added by the AIO utility. Sorry for the trouble. It's amazing what gets missed at the end of the day and you're lacking sleep.

**Comment by bassmannate** [comment]
> How did you install docker on omv? Is docker installed natively or inside a VM?

One more thing. I noticed that after the manual back up, it did not automatically restart containers. Is this something I need to worry about with the automatic backups? Or do I need to make a script to ensure that everything automatically starts after the backup is complete?

**Reply by szaimen** [reply]
The daily backup will also auzomatically start the containers back up again. So nothing to worry about.
